cmd/cli/app/provider/provider_get.go

```diff
@@ -81,6 +81,7 @@ func GetProviderCommand(ctx context.Context, cmd *cobra.Command, _ []string, con
 		impls := getImplementsAsStrings(p)
 		afs := getAuthFlowsAsStrings(p)
 
+		t.AddRow("ID", p.GetId())
 		t.AddRow("Name", p.GetName())
 		t.AddRow("Project", p.GetProject())
 		t.AddRow("Version", p.GetVersion())
```

Comment:
This is fine for now, but these wide tables end up being a bit ugly, so we'll probably (at some future point) need to figure out how to collapse them more elgantly.  (And a 36-character wide UUID doesn't help, but it's not the only contributor)